test/controllers/to_do_items_controller_test.rb

```diff
@@ -30,4 +30,24 @@ class ToDoItemsControllerTest < ActionController::TestCase
     item.reload
     assert item.done
   end
+
+  test "should change position of item on moving" do
+    first_item = items(:first)
+    new_item_position = items(:fourth).position
+
+    xhr :put, :move, {id: first_item.id, position: new_item_position}
+
+    first_item.reload
+    assert_equal new_item_position, first_item.position
```

Comment:
@vlasiak I suggest simply to use `4` value instead of `new_item_position` variable.